Formula/swi-prolog.rb

```diff
@@ -17,35 +16,25 @@ class SwiProlog < Formula
     depends_on "cmake" => :build
```

Comment:
Remove `cmake` here

Reply:
But cmake is a build-time dependency since 7.7.20, `configure` script is no longer provided.

Anything obvious that I'm missing?

Reply:
It needed to be removed from the `head` block. I've done it now.

Reply:
Oh sorry I didn't notice that. Thanks for updating it.